DashAI/front/src/components/experiments/PrepareDatasetStep.jsx

```diff
@@ -58,7 +64,31 @@ function PrepareDatasetStep({ newExp, setNewExp, setNextEnabled }) {
         console.error("Unknown Error", error.message);
       }
     } finally {
-      setLoading(false);
+      setInfoLoading(false);
```

Comment:
What is the advantage of having independent loadings?
I am a bit worried that they may get out of sync and some of them will be left loading infinitely.